demoapp/demo_app.py

```diff
@@ -23,6 +23,10 @@
 from tagging import *
 
 
+# This code only implements text summarization, category selection and tagging for all the bills (no MGL sections data is available for ~1300 bills )
```

Comment:
Nice, thanks for explaining. Please also move this to line  1 (doclevel comment)